src/banner.html

```diff
@@ -11,7 +11,7 @@
 
   <div class="nhsuk-success-banner" id="nhsuk-cookie-confirmation-banner" style="display:none;" role="banner">
     <div class="nhsuk-width-container">
-      <p>Your cookie settings have been saved. <a href="/our-policies/cookies-policy/" style="color: #ffffff;">Change your cookie settings.</a></p>
+      <p>Your cookie settings have been saved. <a href=BANNER_LINK style="color: #ffffff;">Change your cookie settings.</a></p>
```

Comment:
probably need double quotes around `BANNER_LINK`?

Reply:
upon further inspection it seems that you can't access process.env from html anyway. It would need a HTMLwebpackplugin, like here - https://stackoverflow.com/questions/49375867/how-do-you-reference-a-process-env-variable-in-html-script-src-react

Worth it?

Reply:
Then how did this work!?
Maybe we should change our html into a proper template language anyway (e.g nunjucks)